components/input/input.css

```diff
@@ -0,0 +1,60 @@
+@import "/common.css";
+
+.input-container {
+  display: flex;
+  max-width: fit-content;
+  flex-direction: column;
+}
+
+.input-title {
+  font-size: 2rem;
+  font-weight: 600;
+  margin-bottom: 1rem;
+}
+textarea {
+  resize: none;
+}
+
+textarea {
+  margin-bottom: 2rem;
+  padding: 1rem;
+  margin-left: 1rem;
+  max-height: 10rem;
+}
+
+.form-login {
+  background: var(--secondary-color);
+  border-radius: 1rem;
+  box-shadow: 0.1rem 0.1rem 0.2rem var(--secondary-color);
+  max-width: 30rem;
```

Comment:
Again here I would suggest use px value for anything less than 10px here it seems its less than 5px so you must use px values

Reply:
done